test_opensearchpy/test_server/test_rest_api_spec.py

```diff
@@ -72,7 +72,6 @@
 # broken YAML tests on some releases
 SKIP_TESTS = {
     # Warning about date_histogram.interval deprecation is raised randomly
-    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/search_pipeline/10_basic",
     "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/pit/10_basic",
```

Comment:
Please remove the change `OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/search_pipeline/10_basic` to avoid blocking this PR, as two tests are currently failing. 

It will be handled in #726 . Integration tests with OpenSearch 2.7 and 2.8 are failing due to a server bug, and I'll raise an issue in the appropriate repo.